### ISSUE TITLE
Проведение инвентаризации с помощью терминала сбора данных

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1465]
Originally reported on Google Code with ID 1465

```
Сделать модуль проведения инвентаризации с использованием терминала сбора
данных - Inventory 2295, основные операции:

* выгрузка остатков склада в терминал,

* подсчет товаров терминалом,

* выгрузка товаров в Гедымин,

* печать ведомости инвентаризации где "по данным бухучета" выводятся данные
из склада до проведения инвентаризации и "фактическое" по результатам
подсчета, с заполнением поля "недостача", "излишки", данную ведомость будут
подписывать члены инвентаризационной комиссии.
```

Reported by `ivan.nozhko` on 2009-07-06 14:37:14

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2009-07-09 13:28:58
- **Labels added**: Cash, Depot

**Comment by gsbelarus** [comment]
```
По поводу отчетов:
отличаются ли они от тех, что уже есть (для проведения инвентаризации вручную)?
```

Reported by `alexandra.gsoftware` on 2009-07-15 14:22:35

**Comment by gsbelarus** [comment]
```
Есть, замечания по этому отчету, инженер курирующий этот вопрос выйдет из отпуска и
прокомментирует вопрос во вторник.
```

Reported by `sjuchniewicz` on 2009-07-16 14:09:58

**Comment by gsbelarus** [comment]
```
Отчеты проверены. Замечаний нет. При проведении инвентаризации с помощью терминалов(3
шт. у организации) отдельный отчет по инвентаризации делать не нужно. Коррективы в
отчеты могут быть внесены клиентом в рабочем процессе.
```

Reported by `ivan.nozhko` on 2009-07-21 07:55:53

**Comment by gsbelarus** [comment]
```
отправила настройку и описание на e-mail.
```

Reported by `alexandra.gsoftware` on 2009-07-21 11:41:28

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2009-10-08 11:13:08
- **Labels added**: Garant

**Comment by gsbelarus** [comment]
```
Объединяю с другой задачей по терминалу
```

Reported by `alexandra.gsoftware` on 2010-03-30 17:50:44
- **Status changed**: `Duplicate`
- **Merged into**: #1577
